client/src/api.ts

```diff
@@ -197,3 +236,26 @@ export async function createEvent(event: PoapEvent) {
     headers: { 'Content-Type': 'application/json' },
   });
 }
+
+export async function getSigners(): Promise<AdminAddress[]> {
+  return fetchJson(`${API_BASE}/signers`);
+}
+
+export function setSigner(id: number, gasPrice: string): Promise<any> {
+  return secureFetchNoResponse(`${API_BASE}/signers/${id}`, {
+    method: 'PUT',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({gas_price: gasPrice})
+  });
+}
+
+export function getTransactions(limit: number, offset: number): Promise<any> {
```

Comment:
I guess we should add the `Transaction` type in the `getTransactions`


```typescript
export function getTransactions(limit: number, offset: number): Promise<Transaction[]> {
```

Reply:
Actually, is `Promise<PaginatedTransactions>` but I got the point. Thanks!